cws-installer/src/main/java/jpl/cws/task/CwsInstaller.java

```diff
@@ -1812,13 +1812,15 @@ private static int validateElasticsearch() {
 		print("checking that user provided Elasticsearch (" + elasticsearch_host + ":" + elasticsearch_port + ") is running...");
 
 		try {
-			String[] cmdArray = new String[] {"curl", "--fail", "http://" + elasticsearch_host + ":" + elasticsearch_port + "/_cluster/health"};
+			String[] cmdArray = new String[] {"curl", "-Lkv", "--fail", "http://" + elasticsearch_host + ":" + elasticsearch_port + "/_cluster/health"};
 
 			if (elasticsearch_use_auth.equalsIgnoreCase("Y")) {
 				// Add auth to curl
-				cmdArray = new String[] {"curl", "--fail", "-u", elasticsearch_username + ":" + elasticsearch_password, "https://" + elasticsearch_host + ":" + elasticsearch_port + "/_cluster/health"};
+				cmdArray = new String[] {"curl", "-Lkv", "--fail", "-u", elasticsearch_username + ":" + elasticsearch_password, "https://" + elasticsearch_host + ":" + elasticsearch_port + "/_cluster/health"};
```

Comment:
@LucaCinquini Is `-k` the only necessary option to add here? I'm not sure it's necessary to follow redirects & add verbosity to the response. 

The [docs for -k](https://curl.se/docs/manpage.html#-k) would suggest that it's the only necessary option to ignore invalid / unrecognized certs.